blog/2015/04/workshops-for-companies.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+root: ../../..
+author: Matt Davis
+title: "Workshops for Companies"
+date: 2015-04-09
+time: "16:00:00"
+category: ["Workshops"]
+---
+<!-- start excerpt -->
+<p>
+  At our March 12, 2015 meeting, the Software Carpentry Foundation Steering
+  Committee discussed whether and under what terms to provide workshops to
+  for-profit corporations. There were no objections to the idea of doing
+  workshops for corporations, but it is something new to Software Carpentry.
+  We've decided to run a pilot program encompassing five corporate workshops
+  so we can learn more about working with corporations. The pilot program
+  will allow us to gauge corporate interest and collect feedback from
+  instructors, coordinators, and others about how things go.
+</p>
+<!-- end excerpt -->
+<p>
+  During the pilot, workshops will be run on these terms:
```

Comment:
I would simplify this line to "Workshops at corporations will be run on these terms:".